Vlv2TbltDevicePkg/Library/CpuIA32Lib/CpuIA32Lib.inf

```diff
@@ -1,46 +1,57 @@
-#
-#
-# Copyright (c)  1999  - 2014, Intel Corporation. All rights reserved
-#                                                                                  

-# This program and the accompanying materials are licensed and made available under

-# the terms and conditions of the BSD License that accompanies this distribution.  

-# The full text of the license may be found at                                     

-# http://opensource.org/licenses/bsd-license.php.                                  

-#                                                                                  

-# THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,            

-# WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.    

-#                                                                                  

-#
-#
-#   Module Name:
-#
-#     CpuIA32Lib.inf
-#
-#   Abstract:
-#
-#     Component description file for the Cpu IA32 library.
-#
-#--*/
-
-[defines]
-  INF_VERSION                    = 0x00010005
-  BASE_NAME                      = CpuIA32Lib
-  FILE_GUID                      = 98546178-64F1-4d2e-814F-6BF963DB7930
-  MODULE_TYPE                    = BASE
-  VERSION_STRING                 = 1.0
-  LIBRARY_CLASS                  = CpuIA32Lib
-  PI_SPECIFICATION_VERSION       = 0x0001000A
-
-[Sources]
-  EfiCpuVersion.c
-
-[Sources.IA32]
-  IA32/CpuIA32.c
-
-[Sources.X64]
-  X64/Cpu.asm
-  X64/Cpu.S
-
-[Packages]
-  MdePkg/MdePkg.dec
-  Vlv2TbltDevicePkg/PlatformPkg.dec
+#
+#
+# Copyright (c)  1999  - 2014, Intel Corporation. All rights reserved
+#                                                                                  
+
+# This program and the accompanying materials are licensed and made available under
+
+# the terms and conditions of the BSD License that accompanies this distribution.  
+
+# The full text of the license may be found at                                     
+
+# http://opensource.org/licenses/bsd-license.php.                                  
+
+#                                                                                  
+
+# THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,            
+
+# WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.    
+
+#                                                                                  
```

Comment:
@krystian-hebel those comments look like a mess, please correct this file so it will include only necessary changes.

Reply:
@pietrushnic original line endings are a mess. There are some `\n`, `\r\n`, even `\r\r\n`, all in one file. It is hard to edit those files without editor automatically fixing at least `\r\r\n`.